### ISSUE TITLE
Merge 2 workspaces

### ISSUE THREAD

**Issue by Max094Reikeb** [Team-MCToolkit/MCToolkit#44]
**Describe the feature**

We should have the ability to merge 2 workspaces together.

**How would you implement it?**

It could be an option and we could select the other workspace.

**Comment by SlayDigger** [comment]
Yes, this would be an awesome feature! A button to press that says merge workspace, then choose one of the other workspaces in users/user123/MCToolkitWorkspaces
